Kickstarter-iOS/Views/Controllers/LoginToutViewController.swift

```diff
@@ -115,6 +117,17 @@ internal final class LoginToutViewController: UIViewController, MFMailComposeVie
         Strings.login_tout_back_intent_traditional_login_button()
       }
 
+    _ = self.fbDisclaimerStackView
+      |> baseStackViewStyle
+      |> UIStackView.lens.spacing .~ Styles.gridHalf(1)
+
+    _ = self.getNotifiedLabel
+      |> baseLabelStyle
+      |> UILabel.lens.font %~~ { _, label in
+        label.traitCollection.isRegularRegular ? .ksr_headline(size: 15.0) : .ksr_headline(size: 12.0)
```

Comment:
Can we try to use a font that doesn't need a size override? So maybe `.ksr_subhead().bolded` and `.ksr_caption1().bolded`?